checkov/terraform/checks/resource/kubernetes/PrivilegedContainerPSP.py

```diff
@@ -0,0 +1,31 @@
+from checkov.common.models.enums import CheckCategories, CheckResult
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+
+
+class PrivilegedContainersPSP(BaseResourceCheck):
+
+    def __init__(self):
+        # CIS-1.3 1.7.1
+        # CIS-1.5 5.2.1
+        name = "Do not admit privileged containers"
+        id = "CKV_K8S_2"
+
+        supported_resources = ['kubernetes_pod']
+        categories = [CheckCategories.GENERAL_SECURITY]
+        super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources)
+
+    def scan_resource_conf(self, conf) -> CheckResult:
+        count = 0
+        spec = conf['spec'][0]
+        containers = spec.get("container")
+        for container in containers:
+            if container.get("security_context"):
+                context = container.get("security_context")[0]
+                if context.get("privileged") == [True]:
+                    self.evaluated_keys = [f'spec/[0]/container/[{count}]/security_context/[0]/privileged']
+                    return CheckResult.FAILED
+            count += 1
+        return CheckResult.PASSED
```

Comment:
```suggestion
        spec = conf['spec'][0]
        containers = spec.get("container")
        for idx, container in enumerate(containers):
            if container.get("security_context"):
                context = container.get("security_context")[0]
                if context.get("privileged") == [True]:
                    self.evaluated_keys = [f'spec/[0]/container/[{idx}]/security_context/[0]/privileged']
                    return CheckResult.FAILED
        return CheckResult.PASSED
```
you can leverage `enumerate()` to get the index of the list item